src/Domain/Creators/Interval.hpp

```diff
@@ -51,6 +57,17 @@ class Interval : public DomainCreator<1> {
     static constexpr Options::String help = {
         "Sequence of [x] for upper bounds."};
   };
+  struct Distribution {
+    using type = CoordinateMaps::Distribution;
+    static constexpr Options::String help = {"Distribution of grid points"};
+  };
+  struct Singularity {
+    using type = Options::Auto<double, Options::AutoLabel::None>;
+    static constexpr Options::String help = {
+        "Position of coordinate singularity. Must be outside the domain. "
+        "Required for 'Logarithmic' and 'Inverse' grid point distributions. "
+        "Set to 'None' otherwise."};
```

Comment:
Can you give an example of what "Singularity" could be? I have no idea what this means for Inverse, I assume for Log it's `Log(x + singularity)`

Reply:
Here's the new help string (squashed in):

```
Position of coordinate singularity. Must be outside the domain.
Required for 'Logarithmic' and 'Inverse' grid point distributions.
Set to 'None' otherwise.
A singularity position close to the lower or upper bound of the
interval leads to very small grid spacing near that end, and
placing the singularity further away from the domain increases the
grid spacing. See the documentation of
'domain::CoordinateMap::Distribution' for details.
```

https://spectre-code.org/classdomain_1_1CoordinateMaps_1_1Interval.html